app/src/main/java/org/simple/clinic/patient/onlinelookup/api/RecordRetention.kt

```diff
@@ -12,4 +15,12 @@ data class RecordRetention(
     @Json(name = "duration_seconds")
     @SecondsDuration
     val retainFor: Duration?
-)
+) {
+
+  fun computeRetainUntilTimestamp(instant: Instant): Instant? {
+    return when (type) {
+      Temporary -> instant + retainFor!!
```

Comment:
Huh, didn't knew they had operators support. TIL

Reply:
Yeah, this was new to me too. I think the Kotlin compiler handles this automatically if they see a `plus` or `minus` method name.